From 4a752bc3aa85d6eef7e157f9d830f24e264b8227 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 5 Jun 2018 13:49:09 +0800
Subject: [PATCH 3910/5242] MLK-18504: ARM64: dts: Support ASRC P2P with ESAI
 device

commit  be49848de6abd01d235b86437a786c0be6b4f835 from
https://source.codeaurora.org/external/imx/linux-imx.git

Support ASRC P2P with ESAI to show the multichannel capibility.
And make this setting align with other platform like imx6

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Reviewed-by: Viorel Suman <viorel.suman@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts  |    2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 609a2ce..7b467cb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -89,7 +89,6 @@
 		model = "wm8960-audio";
 		cpu-dai = <&sai1>;
 		audio-codec = <&wm8960>;
-		asrc-controller = <&asrc0>;
 		codec-master;
 		/*
 		 * hp-det = <hp-det-pin hp-det-polarity>;
@@ -123,6 +122,7 @@
 		model = "imx-cs42888";
 		esai-controller = <&esai0>;
 		audio-codec = <&cs42888>;
+		asrc-controller = <&asrc0>;
 		status = "okay";
 	};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 8a5bb57..5006b53 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -92,7 +92,6 @@
 		model = "wm8960-audio";
 		cpu-dai = <&sai1>;
 		audio-codec = <&wm8960>;
-		asrc-controller = <&asrc0>;
 		codec-master;
 		/*
 		 * hp-det = <hp-det-pin hp-det-polarity>;
@@ -135,6 +134,7 @@
 		model = "imx-cs42888";
 		esai-controller = <&esai0>;
 		audio-codec = <&cs42888>;
+		asrc-controller = <&asrc0>;
 		status = "okay";
 	};
 };
-- 
1.7.9.5

